vars.yml

```diff
@@ -31,4 +32,5 @@ devshop_cli_path: /usr/local/bin/devshop
 vagrant_development: true
 vagrant_private_network_ip: 10.10.10.10
 vagrant_install_script: install.sh
+vagrant_install_script_args: [--playbook=/vagrant, --server-webserver=nginx]
```

Comment:
I think we can remove this.

See my comment at https://github.com/opendevshop/devshop/pull/18/files#r39530171